probe_scraper/parsers/histograms.py

```diff
@@ -37,8 +37,8 @@ def extract_histogram_data(histogram, version):
         value = None
         if hasattr(histogram, source_field):
             value = getattr(histogram, source_field)()
-        elif source_field in histogram._definition:
-            value = histogram._definition.get(source_field)
+        elif hasattr(histogram, "_" + source_field):
+            value = getattr(histogram, source_field)
```

Comment:
`_definition` is no longer available, but the fields being pulled from it here are now available as separate private properties